lib/utils.pm

```diff
@@ -85,7 +85,8 @@ sub wait_boot {
         }
         return;
     }
-    else {
+    # On Xen PV we don't see a Grub menu
+    elsif (!(check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'linux'))) {
```

Comment:
Added explicit `else {}` recording a soft failure. Though on Xen PV we should be in theory able to `wait_serial` for "GRUB", or something like that, but from some reason atm I don't have serial nor VNC after reboot there..